app/controllers/collections_controller.rb

```diff
@@ -13,6 +13,8 @@ def new
 
     @form = CollectionForm.new(collection)
     @form.prepopulate!
+
+    add_breadcrumb title: 'Create', confirm: true
```

Comment:
This should be "New collection" to be consistent with the design for items: https://projects.invisionapp.com/share/EQXC9CLKCR2#/screens/428088403

Reply:
Note that the design doc actually says "Create Collection" in the breadcrumb nav for this page, but agreed consistency with the Work makes sense and will change to that (unless @astridu disagrees?).  See https://docs.google.com/document/d/1fcr2DYo7OrX-qTdeUOTWKSS1rlq_47sehESmbvgUsrk/edit#heading=h.1mct3tk7ytls